[MFirewall] Managed lists (#10248)
cloudflare/cloudflare-docs: content/magic-firewall/how-to/use-rules-list.md

    "enabled": true
}'
```

## Managed lists

> {{<Aside type="note">}}Available for customers with a Magic Firewall Advanced plan{{</Aside>}}

Reviewer: ```suggestion
{{<Aside type="note">}}Available for customers with a Magic Firewall Advanced plan.{{</Aside>}}
```

> You can  create rules with managed lists. Managed lists are [lists of IP addresses](/fundamentals/global-configurations/lists/ip-lists/#managed-ip-lists) maintained by Cloudflare and updated frequently.

Reviewer: ```suggestion
You can create rules with managed lists. Managed IP lists are [lists of IP addresses](/fundamentals/global-configurations/lists/ip-lists/#managed-ip-lists) maintained by Cloudflare and updated frequently.
```
I would call it "Managed IP lists" because we may have other types of managed lists. Also, I've been using "Managed IP Lists" (with `Lists` capitalized), but we can change this.

Author: I think we can leave it as you have. I'm going to change it on my side. Thanks!

> You can access these managed lists when you create rules with either _IP destination address_ or _IP source address_ in the **Field** drop-down, and _is in list_ or _is not in list_ from the **Operator** drop-down.

Reviewer: ```suggestion
You can access these managed lists when you create rules with either _IP destination address_ or _IP source address_ in the **Field** dropdown, and _is in list_ or _is not in list_ in the **Operator** dropdown.
```
I would use either `<value> in ...` or `select <value> from ...`.

For example:

Field                    | Operator     | Value
------------------------ | ------------ | ---

> _IP Destination Address_ | _is in list_ | _Anonymizers_

Reviewer: ```suggestion
_IP destination address_ | _is in list_ | _Anonymizers_
```
